pkg/chrootarchive/archive_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
```

Comment:
This build tag isn't needed because this gets built only on windows due to having `_windows` in the file's name.